ckanext/datagovuk/public/datagovuk.css

```diff
@@ -78,3 +79,41 @@ a.btn .icon-briefcase {
 label:after {
   content: "";
 }
+
+@media (min-width: 768px) {
+  .hero {
+    background: #128400;
+  }
+}
+
+.ie8 .hero,
+.ie7 .hero {
+  background: #128400;
+}
+
+.home-content {
+  color: #FFFFFF;
```

Comment:
It's usually less cognitive load to use `white` and `black` or `#FFF` and `#000` for `#FFFFFF` and `#000000` but I don't mind much.